applyHint.sh

```diff
@@ -0,0 +1,22 @@
+function applyHint {
+for fileName in `find $1 -iname "*.hs"`
+do
+    # recursive because sometimes we have hints inside of other hints
+    for i in {1..3}
+    do
+        if hlint --only "Redundant bracket" $fileName | grep "No hints"
+        then
+            echo "No hints found: loop $i; file: $fileName"
+            break
+        else
+            echo "Applying redundant bracket hints automatically: loop $i; file: $fileName"
+            hlint --refactor --only "Redundant bracket" $fileName > $fileName.tmp
```

Comment:
`hlint --refactor --only "Redundant bracket" $fileName > $fileName`
this command did not work for me, so I've added tmp file. Please suggest how to avoid creating extra file